account-kit/rn-signer/example/src/App.tsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/extensions */
+import { NativeTEKStamper } from "@account-kit/react-native-signer";
+import type { User } from "@account-kit/signer";
+import { useEffect, useState } from "react";
+import { Button, StyleSheet, Text, TextInput, View } from "react-native";
+import { SignerClient } from "./signer";
+
+export default function App() {
+  const [publicKey, setPublicKey] = useState<string | null>(null);
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'publicKey' is assigned a value but never used.
<!-- __reviewdog__:ChAyMjRlMDcxZjFkN2FiY2M3EgZyZGpzb24= -->